lib/Runtime/PlatformAgnostic/ChakraPlatform.h

```diff
@@ -5,3 +5,8 @@
 #pragma once
 
 #include "UnicodeText.h"
+#include "Platform/Common/DateTime.h"
+
+#if !defined(_WIN32) && defined(DEBUG)
```

Comment:
It's not. It provides a definition to `signal` for debug builds. Instead of adding it to PAL, I thought PlatformAgnostic is the right place.